libs/utils/utils.js

```diff
@@ -151,7 +151,8 @@ export const [setConfig, getConfig] = (() => {
       config.autoBlocks = conf.autoBlocks ? [...AUTO_BLOCKS, ...conf.autoBlocks] : AUTO_BLOCKS;
       document.documentElement.setAttribute('lang', config.locale.ietf);
       try {
-        document.documentElement.setAttribute('dir', (new Intl.Locale(config.locale.ietf)).textInfo.direction);
+        const contentDir = getMetadata('content-direction');
+        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || (new Intl.Locale(config.locale.ietf)).textInfo.direction);
```

Comment:
// @TsayAdobe Either you can add the contentDir based the [discussion](https://github.com/orgs/adobecom/discussions/746#discussioncomment-5981042) and close https://github.com/adobecom/milo/pull/773/files or we use this PR. I'm fine with either. 
```suggestion
        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || 'ltr')
```